zerver/models.py

```diff
@@ -1854,3 +1855,13 @@ def get_bot_services(user_profile_id):
 def get_service_profile(user_profile_id, service_name):
     # type: (str, str) -> Service
     return Service.objects.get(user_profile__id=user_profile_id, name=service_name)
+
+def get_service_dicts_for_bots(user_profile_id):
+    # type: (int) -> List[Dict[str, Any]]
+    services = get_bot_services(str(user_profile_id))
+    service_dicts = ([{'name': service.name,
+                       'base_url': service.base_url,
+                       'interface': service.interface,
+                       }
+                      for service in services])
+    return service_dicts
```

Comment:
@timabbott Thanks for the heads up! My code is more or less a copy of this PR. 
I am not able to come up with any alternate than this, any suggestions from your side?

One approach that I am able to think of from our previous discussions is that since initially we plan to link only 1 service to a bot, we can directly store that one service as a field in "UserProfile". But that doesn't seem good to me as:
* It will restrict our future exploration into services related to bots.
* Creating an altogether separate field for "services" seems unnecessary.

What are your thoughts?